warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1061,6 +1093,18 @@ protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, Shard
             }
         }
         
+        // apply the node transform rules
+        if (!getTransformRules().isEmpty()) {
+            stopwatch = timers.newStartedStopwatch("DefaultQueryPlanner - Apply Node Transform Rules");
+            
+            queryTree = applyNodeTransformRules(queryTree, getTransformRules(), config);
+            
+            if (log.isDebugEnabled()) {
+                logQuery(queryTree, "Query after function index queries were expanded:");
+            }
+            stopwatch.stop();
+        }
```

Comment:
same exact code block 3 times.  same timer message, same log message.  Consider making its own method?  Or at least give some indication of which iteration we're on for logging.

Reply:
that is fair....will do